packages/vit-s/src/mui-utils.js

```diff
@@ -0,0 +1,49 @@
+// This function has been copied from MUI v4
+// with logic related to randomness / random numbers removed.
+// Reference: https://github.com/mui/material-ui/blob/0adada09/packages/mui-styles/src/createGenerateClassName/createGenerateClassName.js#L29
+/**
+ * This is the list of the style rule name we use as drop in replacement for the built-in
+ * pseudo classes (:checked, :disabled, :focused, etc.).
+ *
+ * Why do they exist in the first place?
+ * These classes are used at a specificity of 2.
+ * It allows them to override previously defined styles as well as
+ * being untouched by simple user overrides.
+ */
+const stateClasses = [
+  'checked',
+  'disabled',
+  'error',
+  'focused',
+  'focusVisible',
+  'required',
+  'expanded',
+  'selected',
+];
+
+const seedPrefix = 'vit-';
+const disableGlobal = false;
```

Comment:
How can this be changed?  What is it for?

Reply:
I am not sure, I have never really understood the docs about it https://mui.com/system/styles/api/#creategenerateclassname-options-class-name-generator - It is not necessary so I will remove it